### PR TITLE
JMLL-845 Update UK SBS API Leaving submission data

### PR DESCRIPTION
**Pull request by jafacakes2011**

### CODE REVIEW

File: core/uksbs/utils.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    leaver_full_name = uksbs_leaver["full_name"]
+
+    leaver_contact: ServiceRequestDataContact = {
+        "contactNumber": "ORACLE_ID",  # TOOD
+        "contactType": "EMPLOYEE",
+        "contactTypePoint": "EMAIL",
+        "contactPrimaryFlag": "N",
+    }
+    line_manager_contact: ServiceRequestDataContact = {
+        "contactNumber": "ORACLE_ID",  # TOOD
```
Comment (rossmiller) [inline core/uksbs/utils.py:111]: Does that TODO need to be tackled?

**Review by rossmiller — APPROVED** [review]
A couple of very minor points to address. As usual, outstanding work!!

File: core/uksbs/utils.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            uksbs_leaver_manager = ulm
+            break
+
+    if not uksbs_leaver_manager:
+        raise Exception("Could not find line manager in UKSBS hierarchy")
+
+    leaver_full_name = uksbs_leaver["full_name"]
+
+    leaver_contact: ServiceRequestDataContact = {
+        "contactNumber": "ORACLE_ID",  # TOOD
```
[the remark this answers is not in the document]
Comment (jafacakes2011) [reply core/uksbs/utils.py:105]: Yes, which field in `ActivityStreamStaffSSOUser` can we consider the Oracle ID?

File: core/uksbs/utils.py (hunk 1 above)
Comment (jafacakes2011) [reply core/uksbs/utils.py:111]: Yes see previous

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**